lib/iris/tests/test_util.py

```diff
@@ -90,43 +89,36 @@ def test_oversize_string(self):
         result = iris.util.clip_string(self.test_string, clip_length, self.rider)
 
         # Check the length is between what we requested ( + rider length) and the length of the original string
-        self.assertTrue(
-            clip_length + len(self.rider) <= len(result) < len(self.test_string),
-            "String was not clipped.",
-        )
+        assert (
+            clip_length + len(self.rider) <= len(result) < len(self.test_string)
+        ), "String was not clipped."
 
         # Also test the rider was added
-        self.assertTrue(
-            self.rider in result,
-            "Rider was not added to the string when it should have been.",
-        )
+        assert (
+            self.rider in result
+        ), "Rider was not added to the string when it should have been."
 
     def test_undersize_string(self):
         # Test with a clip length that is longer than the string
 
         clip_length = 10999
         result = iris.util.clip_string(self.test_string, clip_length, self.rider)
-        self.assertEqual(
-            len(result),
-            len(self.test_string),
-            "String was clipped when it should not have been.",
-        )
+        assert len(result) == len(
```

Comment:
Most asserts with a message attached are wrapped in brackets, might be worth keeping it consistent (unless I've missed something)!